app/src/main/java/org/p2p/wallet/utils/BiometricPromptWrapper.kt

```diff
@@ -14,19 +14,24 @@ class BiometricPromptWrapper(
     @StringRes private val descriptionRes: Int = R.string.auth_biometric_question,
     @StringRes private val negativeRes: Int = R.string.common_cancel,
     private val onError: ((String?) -> Unit)? = null,
+    private val onLockout: ((String?) -> Unit)? = null,
     private val onSuccess: (Cipher) -> Unit
 ) {
 
     private val biometricCallback = object : BiometricPrompt.AuthenticationCallback() {
 
         override fun onAuthenticationError(errorCode: Int, errString: CharSequence) {
-            onError?.invoke(
-                errString.takeIf {
-                    errorCode != BiometricPrompt.ERROR_USER_CANCELED &&
-                        errorCode != BiometricPrompt.ERROR_NEGATIVE_BUTTON
-                }
-                    ?.toString()
-            )
+            if (errorCode == BiometricPrompt.ERROR_LOCKOUT) {
```

Comment:
are we logging stuff here? smth like Timber.i?

Reply:
no logs
why do we need this?